core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java

```diff
@@ -13,43 +13,29 @@
 
 import alluxio.exception.PageNotFoundException;
 
+import java.util.HashSet;
+import java.util.Set;
+
 /**
  * The default implementation of a metadata store for pages stored in cache.
  */
 public class DefaultMetaStore implements MetaStore {
+  private final Set<PageId> mPageMap = new HashSet<>();
```

Comment:
Is this implementation actually exactly the same as the `HashSetMetaStore` in the test? If so maybe we only need 1 of these?

Reply:
looks like so. Then we should dedup and keep one impl `DefaultMetaStore`.

Reply:
We can reuse the `DefaultMetaStore` in tests for now, but it is likely we will have to reintroduce the test implementation as the `DefaultMetaStore` become more complex.

Reply:
if that's the case we can keep them separate.